Correct some mistakes in the README

Avaq opened the pull request.

Avaq committed:
Fix the example in the README

Avaq committed:
Fix the signature of hookAll

Co-Authored-By: Onur Gunduz <ogunduz@gmail.com>

dicearr commented on index.mjs:
```diff
@@ -26,11 +26,9 @@
 //. const withRedis = hook (acquireRedis) (closeConnection);
 //.
-//. const withServices = hookAll ([withPostgres, withRedis]);
+//. const app = runHook (hookAll ([withPostgres, withRedis]));
```
```suggestion
//. const withServices = runHook (hookAll ([withPostgres, withRedis]));
```

dicearr commented on index.mjs:
```diff
@@ -26,11 +26,9 @@
-//. const app = runHook (([postgres, redis]) => {/*...*/});
-//.
-//. fork (console.error) (console.log) (app (withServices));
+//. fork (console.error) (console.log) (app (([postgres, redis]) => {}));
```
```suggestion
//. fork (console.error) (console.log) (withServices (([postgres, redis]) => {}));
```

dicearr reviewed (dismissed):
I think `withServices` is more descriptive. However, it is a personal preference.

Avaq committed:
Improve the example in the README

- Use a clearer naming convention
- Be more explicit about how the services are consumed

Co-Authored-By: Diego Ceresuela <dcerearr@gmail.com>

Avaq committed:
Make code style in the README more consistent

dicearr commented on index.mjs:
```diff
@@ -8,29 +8,32 @@
 //.
 //. const acquireRedis = (
-//.   node(done => require ('imaginary-redis') .connect (done))
+//.   node (done => require ('imaginary-redis') .connect (done))
```
```suggestion
//.   node (done => require ('imaginary-redis').connect (done))
```

dicearr commented on index.mjs:
```diff
@@ -8,29 +8,32 @@
 //. import {hook, hookAll, runHook} from 'fluture-hooks';
 //.
 //. const acquirePostgres = (
 //.   node (done => require ('imaginary-postgres') .connect (done))
```
```suggestion
//.   node (done => require ('imaginary-postgres').connect (done))
```
